pkg/identity/identity.go

```diff
@@ -76,3 +79,87 @@ func (id *signingIdentity) Credentials() []byte {
 func (id *signingIdentity) Sign(message []byte) ([]byte, error) {
 	return id.sign(message)
 }
+
+func CreateSigner(privateKeyPath, certificatePath, mspID string) (SigningIdentity, error) {
+	priv, err := GetPrivateKey(privateKeyPath)
+	if err != nil {
+		return nil, err
+	}
+
+	cert, _, err := GetCertificate(certificatePath)
+	if err != nil {
+		return nil, err
+	}
+
+	return NewPrivateKeySigningIdentity(mspID, cert, priv)
+}
+
+func GetPrivateKey(f string) (*ecdsa.PrivateKey, error) {
```

Comment:
Restricting this to only ECDSA keys may not be a good idea. If changes to Fabric to [support ed25519](https://github.com/hyperledger/fabric/pull/3343) keys gets merged, this will be less useful

Reply:
I will move this function to ecdsa.go for admin-sdk code. as further considering with 25519 curve support. the NewPrivateKeySigningIdentity function will have a `crypto.PrivateKey` as input parameter to cover 25519.